src/app/components/shared/content-node-share/content-node-share.dialog.html

```diff
@@ -10,7 +10,6 @@ <h1 data-automation-id="adf-share-dialog-title" class="adf-share-link__title">
       <h1 class="adf-share-link__label">{{ 'SHARE.TITLE' | translate }}</h1>
 
       <mat-slide-toggle
-        color="primary"
```

Comment:
The same fix should be applied to the other mat-slide-toggle - the one next to the 'Expires on'